Rakefile

```diff
@@ -15,7 +15,6 @@ end
 Rake::TestTask.new do |t|
   t.libs << "."
   t.warning = true
-  t.verbose = true
```

Comment:
Can we please not change unrelated things in this pull request? If you think the test output is too noisy please open a separate pull request for that, where we can discuss the pros and cons